File: 2/src2/scores2.c
// Generates a bar chart of three scores using an array

#include <cs50.h>
#include <stdio.h>

void chart(int score);

int main(void)
{
    // Get scores from user
    int scores[3];
    for (int i = 0; i < 3; i++)
    {
        scores[i] = get_int("Score %i: ", i + 1);
    }

    // Chart scores
    for (int i = 0; i < 3; i++)
    {
        printf("Score %i: ", i + 1);
        chart(scores[i]);
    }
}

// Generate bar
void chart(int score)
{
    // Output one hash per point
    for (int i = 0; i < score; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: 6/src6/weeks/2/scores2.py
# Generates a bar chart of three scores using a list

from cs50 import get_int


def main():

    # Get scores from user
    scores = []
    for i in range(3):
        scores.append(get_int(f"Score {i + 1}: "))

    # Chart scores
    for i in range(len(scores)):
        print(f"Score {i + 1}: ", end="")
        chart(scores[i])

def chart(score):
    """Generate bar"""

    # Output one hash per point
    for i in range(score):
        print("#", end="")
    print()


if __name__ == "__main__":
    main()
